src/main/java/org/isf/session/LogoutEventListener.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2021 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.session;
+
+import java.awt.AWTEvent;
+import java.awt.event.AWTEventListener;
+import java.awt.event.KeyEvent;
+
+import org.isf.generaldata.GeneralData;
+import org.isf.utils.jobjects.DelayTimerCallback;
+
+public class LogoutEventListener implements DelayTimerCallback, AWTEventListener {
+
+	@Override
+	public void trigger() {
+		boolean isNotSingleUserMode = !GeneralData.getGeneralData().getSINGLEUSER();
```

Comment:
What is single user mode even tested for; looking at the code the `LogoutEventListener` is only active in not in single user mode:
<pre>
			if (!GeneralData.getGeneralData().getSINGLEUSER()) {
				startLogoutTimer(mainMenu);
			}
</pre>